app/javascript/hyacinth_v1/components/projects/enabled_dynamic_fields/EnabledDynamicFieldForm.js

```diff
@@ -201,7 +205,7 @@ const DynamicFieldGroup = (props) => {
 };
 
 const DynamicFieldCategory = (props) => {
```

Comment:
```suggestion
const DynamicFieldCategory = ({ category, enabledFieldDataCallback, disabled }) => {
```
I've been getting to the pattern of prop destructuring lately.  This is a completely optional suggestion, but I wanted to show you an example of what I mean.